File: src/main/java/io/zeitwert/fm/app/adapter/api/rest/dto/HomeOverviewResponse.java

package io.zeitwert.fm.app.adapter.api.rest.dto;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class HomeOverviewResponse {

	private Integer accountId;
	private String accountName;
	private String accountImageUrl;

	private Integer buildingCount;
	private Integer portfolioCount;
	private Integer ratingCount;
	private Integer insuranceValue;
	private Integer timeValue;

	private Integer shortTermRenovationCosts;
	private Integer midTermRenovationCosts;

}

File: fm-domain/src/main/java/io/zeitwert/fm/app/adapter/api/rest/dto/HomeOverviewResponse.kt
package io.zeitwert.fm.app.adapter.api.rest.dto

data class HomeOverviewResponse(
	val accountId: Int,
	val accountName: String,
	val buildingCount: Int,
	val portfolioCount: Int,
	val ratingCount: Int,
	val insuranceValue: Int,
	val timeValue: Int,
	val shortTermRenovationCosts: Int,
	val midTermRenovationCosts: Int,
)
